paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSheet.kt

```diff
@@ -92,13 +106,18 @@ class CustomerSheet @Inject internal constructor(
      * a persisted payment option selection.
      */
     suspend fun retrievePaymentOptionSelection(): CustomerSheetResult = coroutineScope {
+        val request = configureRequest
+            ?: return@coroutineScope CustomerSheetResult.Failed(
```

Comment:
Maybe another option is to block until the merchant calls configure? But this probably feels right, we want to give them fast feedback their integration is wrong.

Or maybe we have separate behavior for livemode vs testmode?

Reply:
We could potentially time them out similar to what we do in `CustomerSheetLoader` when we are waiting for the adapter.